src/hooks/useNameDetails.tsx

```diff
@@ -57,18 +59,37 @@ export const useNameDetails = (name: string) => {
       return t('errors.invalidName')
     }
     if (registrationStatus === 'gracePeriod') {
-      return t('errors.expiringSoon')
+      return `${t('errors.expiringSoonOne')} ${gracePeriodEndDate?.toString()} ${t(
+        'errors.expiringSoonTwo',
+      )}`
```

Comment:
the formatting on this could change depending on the translation, should use translation variable in a single translation